nanoFramework.Json.Test/JsonUnitTests.cs

```diff
@@ -1539,7 +1539,7 @@ public void SerializeCosmosDbObject_02()
         public void SerializeStringWithNewLine_Should_ReturnWithoutNewLine()
         {
             var inputData = "multiline\nstring";
-            var expectedJson = "[\"multiline\nstring\"]";
+            var expectedJson = "\"multiline\nstring\"";
```

Comment:
This should be ``var expectedJson = "\"multiline\\nstring\"";`` (same below)

Reply:
Are you sure, it shouldn't be 
var expectedJson = "\"multiline\\\nstring\"" ?

New code pushed.

Reply:
Well, the JSON output should be ``"multiline\nstring"`` - and unless I misunderstand something here, we have to write this as the C# string definition ``"\"multiline\\nstring\""``. You can actually test this in the Visual Studio C# Interactive toolbar thingy:
![grafik](https://user-images.githubusercontent.com/21185621/179371472-62dff6e3-e46c-4e2c-8b46-a1aa3a5987e8.png)